web/src/features/map/Map.tsx

```diff
@@ -64,8 +64,8 @@ export default function MapPage({ onMapLoad }: MapPageProps): ReactElement {
   const getCo2colorScale = useCo2ColorScale();
   const navigate = useNavigate();
   const theme = useTheme();
-  const currentMode = useAtomValue(productionConsumptionAtom);
-  const mixMode = currentMode === Mode.CONSUMPTION ? 'consumption' : 'production';
+  const mixMode = useAtomValue(productionConsumptionAtom);
+  const isConsumption = mixMode === Mode.CONSUMPTION;
```

Comment:
I'll create a derived atom for this in a follow up PR to avoid duplicating this logic everywhere.